scripts/gitbook.js

```diff
@@ -135,19 +113,25 @@ const fromDir = async (startPath, res = []) => {
 
 /**
  * This script creates a SUMMARY.md file used by GitBook to create its table of contents.
- * We get all the .md files inside the apps and libs folders and sub-folders. When pick by the script,
- * we use the first line of the readme to define the name of the GitBook page.
- * If a README.md is not worth being listed in GitBook, use the HTML comment `<!-- gitbook-ignore -->` on
+ *
+ * We automatically get all *.md files in the different directories listed below.
+ * We use the first line of the readme to define the name of the GitBook page.
+ *
+ * If a README.md is not worth being listed in GitBook, use `<!-- gitbook-ignore -->` on
  * the first line of the file.
  *
- * Below are defined the static pages inside the handbook directory. You can add new pages there.
  * To run this script, do `yarn gitbook`
  */
 const generateSummary = async () => {
-  const apps = await fromDir('./apps')
-  const libs = await fromDir('./libs')
-
-  fs.writeFileSync('SUMMARY.md', content(apps, libs))
+  const [apps, libs, overview, repo, misc] = await Promise.all([
+    await fromDir('./apps'),
+    await fromDir('./libs'),
+    await fromDir('./handbook/technical-overview', [], true),
```

Comment:
Why not just `handbook` instead of listing up the subdirectories?

Reply:
I couldn't find an easy way to organize the different sections to follow this: https://github.com/island-is/island.is/blob/main/scripts/gitbook.js#L41-L59